content/app-dev/create-actor/scaffold.md

```diff
@@ -14,7 +14,7 @@ cargo install cargo-generate
 Once you've installed `cargo-generate`, then we can create a new actor using wasmcloud's [new actor template](https://github.com/wasmcloud/new-actor-template):
 
 ```
-cargo generate --git https://github.com/wasmcloud/new-actor-template
+cargo generate --git https://github.com/wasmcloud/new-actor-template --branch main
```

Comment:
You don't need `branch main` here as long as you're running the newest version of cargo generate.